lib/promoted/ruby/client.rb

```diff
@@ -297,8 +297,12 @@ def deliver_shadow_traffic args, headers
           delivery_request_params = delivery_request_builder.delivery_request_params(should_compact: false)
           delivery_request_params[:client_info][:traffic_type] = Promoted::Ruby::Client::TRAFFIC_TYPE['SHADOW']
 
-          # Call Delivery API async (fire and forget)
-          send_request(delivery_request_params, @delivery_endpoint, @delivery_timeout_millis, @delivery_api_key, headers, true)
+          # Call Delivery API and log/ignore errors.
+          begin
+            send_request(delivery_request_params, @delivery_endpoint, @delivery_timeout_millis, @delivery_api_key, headers, true)
+          rescue StandardError => err
```

Comment:
This is a very blunt error type to rescue; in theory it should only be our own EndpointError.